docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+   cinema:
+       build:
+           context: .
+       ports:
+           - "8000:8000"
+       command: >
+           sh -c "python manage.py wait_for_db &&
+                  python manage.py migrate &&
+                  python manage.py runserver 0.0.0.0:8000"
```

Comment:
Consider using `exec` form for the command to ensure that signals are properly received by the application. For example, use `command: ["sh", "-c", "python manage.py wait_for_db && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]`.